src/main/java/com/vpcomtek/onlinebookstore/model/Book.java

```diff
@@ -7,23 +7,37 @@
 import jakarta.persistence.Id;
 import java.math.BigDecimal;
 import lombok.Data;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
 
 @Entity
 @Data
+@SQLDelete(sql = "UPDATE book SET is_deleted = true WHERE id=?")
+@Where(clause = "is_deleted=false")
+//@Table(name = "book")
```

Comment:
don't forget to remove comments before committing
```suggestion
@SQLDelete(sql = "UPDATE book SET is_deleted = TRUE WHERE id = ?")
@SQLRestriction(clause = "is_deleted = FALSE")
```

Reply:
not fixed